infra/recipes/terraform/azure/environments/container/backend.tf

```diff
@@ -1,3 +0,0 @@
-terraform {
-  backend "azurerm" {}
-}
```

Comment:
Maybe comment it out instead of deleting it. And add a comment saying this is for storing terraform.state file in the cloud.

Reply:
Done